.github/workflows/release.yml

```diff
@@ -399,7 +399,7 @@ jobs:
             });
 
   trigger-ee-extra-pr:
-    needs: verify-s3-download
+    needs: add-extra-tags
```

Comment:
I think the race condition is missing the pushed docker container, right? Those extra tags are like 54.x, which I don't think ee extra cares about.

so this should be

```
  needs: [verify-s3-download, containerize-ee]
```
right?